crates/cxx-qt-lib/build.rs

```diff
@@ -49,6 +49,11 @@ fn write_headers() {
     println!("cargo::rerun-if-changed=include/common.h");
     std::fs::copy("include/common.h", header_dir().join("common.h"))
         .expect("Failed to copy header file!");
+    std::fs::copy(
```

Comment:
you need this line here too, like the above common.h has. Otherwise the cargo won't know to rebuild if the assertion_utils changes.
```
println!("cargo::rerun-if-changed=include/assertion_utils.h");
```